.env.example

```
@@ -22,6 +22,14 @@ LAB_AUTH_ALLOWED_REDIRECT_URIS=
 LAB_GOOGLE_CLIENT_ID=
 LAB_GOOGLE_CLIENT_SECRET=
 
+# ─── Local master logs ────────────────────────────────────────────────────────
+# Optional overrides for the persistent local-master log store and live fanout.
+LAB_LOCAL_LOGS_STORE_PATH=
+LAB_LOCAL_LOGS_RETENTION_DAYS=7
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAB_LOCAL_LOGS_RETENTION_DAYS key should go before the LAB_LOCAL_LOGS_STORE_PATH key

(UnorderedKey)